### PR TITLE
Change the bug report issue template in order to avoid "1 of n tasks"  for the environment section

### PR DESCRIPTION
**Pull request by ceofil**

### CODE REVIEW

**Comment by dyc3** [comment]
I wish github had a better way to review these changes. Could you enable issues on your fork so I can preview the changes?

**Comment by ceofil** [comment]
> I wish github had a better way to review these changes. Could you enable issues on your fork so I can preview the changes?

Me too =))
Also the fact that you have to commit the file in order to see if it works. No preview or anything.
I enabled the issues. :D

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -25,14 +25,16 @@ body:
       4. See error...
   validations:
     required: true
-- type: checkboxes
+- type: dropdown
   attributes:
     label: Environment
-    description: Mark the appropriate boxes. If you don't know if the statement applies, leave it blank.
+    description: Select the appropriate options. If you don't know if the statement
+      applies, leave it blank.
```
Comment (dyc3) [inline .github/ISSUE_TEMPLATE/bug_report.yml:32]:
```suggestion
      applies, leave it unselected.
```

**Review by dyc3 — COMMENTED** [review]
Eh, I don't really like how this doesn't show up as a bullet list when rendered in the issue

Example:
image

**Comment by ceofil** [comment]
> Eh, I don't really like how this doesn't show up as a bullet list when rendered in the issue
> 
> Example: image

There is also this option https://github.com/orgs/community/discussions/4319#discussioncomment-5942069

**Comment by dyc3** [comment]
If you can figure out how to keep those options in a list, and have it not show up with "1 of n tasks" in the github UI, I'll happily merge this, but tbh it's not that big of a deal, and probably not worth the effort.

**Pull request closed without merging**